app/views/transferAgreementAlreadyCompleted.scala.html

```diff
@@ -0,0 +1,104 @@
+@* helper package contains inputRadioGroup therefore exclude from imports to prevent ambiguous import with view of the same name *@
+@import helper.{inputRadioGroup => _, _}
+@import viewsapi.FormFunctions._
+@import java.util.UUID
+
+@(consignmentId: UUID, transferAgreementForm: Form[TransferAgreementData], agreementOptions: Seq[(String, String)])(implicit request: RequestHeader, messages: Messages)
+@main("Transfer agreement") {
+@defining(play.core.PlayVersion.current) { version =>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        @progressIndicator(Messages("transferAgreement.progress"))
+
+        <h1 class="govuk-heading-l">Transfer agreement</h1>
+        <p class="govuk-body">You must confirm all statements before proceeding. If you cannot, please close your browser and contact your transfer advisor.</p>
+        <div class="govuk-warning-text">
+            <span class="govuk-warning-text__icon" aria-hidden="true">!</span> <!-- "aria-hidden" was added by GDS to prevent screen reader from saying "exclamation mark"-->
+            <strong class="govuk-warning-text__text">
+              <span class="govuk-warning-text__assistive">Warning</span>
+                This service is in private beta. During this time, the Transfer Digital Records service can only accept records that are in English, are Public Records and are Crown Copyright. For more information, please see our <a href="@routes.FaqController.faq()#other-languages" target="_blank" rel="noopener noreferrer" class="govuk-link">FAQ (opens in new tab)</a> if you need to transfer records that this service cannot currently handle.
+            </strong>
+        </div>
+        <div id="upload-progress-success" class="success-summary" aria-labelledby="success-summary-title"
+        role="alert" data-module="success-summary" tabindex="-1">
+          <div class="success-summary__body">
+            <h2 class="success-summary__title">You have already confirmed all statements</h2>
+            <p class="govuk-body">Click 'Continue' to proceed with your transfer.</p>
+          </div>
+        </div>
+        @form(
+            routes.UploadController.uploadPage(consignmentId),
+            (Symbol("novalidate"), "")
+        ) {
+        <div class="govuk-form-group">
+            @inputSingleCheckbox(
```

Comment:
Bit worried that we'll have to keep this page manually in sync with the existing transfer agreement form - both things like the headers, and the list of options. Is there a good way to commonise them so that they can pull in the shared text?

Reply:
This is a good concern, I could possibly pass in a variable to the view and add some logic that uses it to either hide or display the "already completed" message and the disabling and ticking of the boxes